lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/collection/includeExcludeWidget.py

```diff
@@ -99,15 +151,109 @@ def getIncludedItems(self):
 
     def getExcludedItems(self):
         return self._exclude.list.items()
-    
+
     def getIncludeAll(self):
         return self._include.cbIncludeAll.isChecked()
 
     def setIncludeAll(self, value: bool):
         self._include.cbIncludeAll.setChecked(value)
 
-    def onIncludeAllToggle(self):
-        self._collection.GetIncludeRootAttr().Set(self._include.cbIncludeAll.isChecked())
+    def updateUI(self):
```

Comment:
Eventually, I expect that we will move all code that directly interact with USD out of the UI code and into a "model" that will also "hold" the DCC-specific code or callbacks. So all code currently in UI classes will move in the future.